src/app/notes/models/note.model.ts

```diff
@@ -23,6 +24,10 @@ export function getNoteFromViewModel(note: NoteViewmodel): Note {
     } as Note
 }
 
+export function isOfTypeOtherOrListing(note: Note) {
```

Comment:
unit tests?

Yes, it's trivial logic, but still logic.

Reply:
80% test coverage, so yes this is too trivial logic imho.